builder/tests/builder_test.go

```diff
@@ -31,6 +35,14 @@ func TestBuilder(t *testing.T) {
 		"/deis/domains",
 		"/deis/services",
 	}
+	setproxy := []byte("HTTP_PROXY=\nhttp_proxy=\n")
+
+        tmpfile,errfile := ioutil.TempFile("/tmp", "deis-test-")
```

Comment:
I run gofmt but I was waiting @carmstrong to help me with the docs, let me do this now